crates/red_knot_python_semantic/src/semantic_index/builder.rs

```diff
@@ -561,6 +575,11 @@ where
                         Some(CurrentAssignment::AnnAssign(ann_assign)) => {
                             self.add_definition(symbol, ann_assign);
                         }
+                        Some(CurrentAssignment::AugAssign(aug_assign)) => {
+                            // TODO(dhruvmanila): For augmented assignment, the target expression
+                            // is also used, so we should record that as a use.
```

Comment:
Is there a reason to leave this as a TODO rather than just doing it in this PR? It doesn't seem like it should be too involved.

But I'm also fine with doing it as a separate follow-up PR.

Reply:
Yeah, it isn't that much involved. I've updated this PR to implement the TODO.